### ISSUE TITLE
Request: Allow Quote bot to embed in the channel where it was called from.

### ISSUE THREAD

**Issue by Al12rs** [RalphORama/QuoteBot#4]
I added this bot to the server looking for a fast way to reference pas messages during conversations but then I found out that it seems everything is bound to a single quote output channel.
I realize that this is probably not what the bot was thought for but given that the infrastructure is already there I wanted to ask if it was possible to add the possibility to print quotes directly in the channel where the request was made. 
Of course you probably don't want to compromise the current functionality so I adding a new command like `quotehere` would probably be the best option.

Basically this would allow big servers with multiple channels to easily reference older messages from different channels. 
I looked briefly at the code and it looked possible but I might be mistaken.
